src/components/Button/Button.css.js

```diff
@@ -185,6 +185,10 @@ export const config = {
   focusOutlineWidth: 2,
```

Comment:
To avoid merge conflicts from https://github.com/helpscout/hsds-react/pull/613, can you move this file to it's own styles folder? ❤️